RELEASE-NOTES.txt

```diff
@@ -1,9 +1,13 @@
+2.22
+-----
+
+
 2.21
 -----
 * [Internal] Added login alert for unverified accounts [#1449](https://github.com/Automattic/simplenote-android/pull/1449)
 * Fixed scroll position of a note with matching keyword in not persistent [#1454](https://github.com/Automattic/simplenote-android/pull/1454)
 * Updated icons [#1458](https://github.com/Automattic/simplenote-android/pull/1458)
-* Fixed editor's background color in landscape mode with dark theme [#1457](https://github.com/Automattic/simplenote-android/pull/1457)
+* Fixed editor's background color in landscape mode for dark theme [#1457](https://github.com/Automattic/simplenote-android/pull/1457)
```

Comment:
@danilo04 I've made a very small change to this release note. Hope that's OK with you.

Reply:
💯 . Thanks!